packages/core/src/transports/initialize.ts

```diff
@@ -7,7 +7,7 @@ import { BeforeSendHook, Transport, TransportItemType, Transports } from './type
 export function initializeTransports(config: Config): Transports {
   const transports: Transport[] = [...config.transports];
 
-  let paused = false;
+  let paused = config.paused ?? false;
```

Comment:
At this point, the config should already contain the default values to avoid nullish coalescing.

Reply:
updated